task/05-regex-tasks.js

```diff
@@ -91,10 +87,9 @@ function getRegexForIPv4() {
  * @return {RegExp}
  */
 function getRegexForSSN() {
-   throw new Error('Not implemented');
+  return /^(?!(000))\d{3}-(?!00)\d{2}-(?!0000)\d{4}$/;
```

Comment:
Some special numbers are never allocated:
Numbers with all zeros in any digit group (000-##-####, ###-00-####, ###-##-0000).
**Numbers with 666 or 900-999 (Individual Taxpayer Identification Number) in the first digit group.**

not correct, highlighted rules are not validated